### PR TITLE
New gateway directory ui

### PR DESCRIPTION
**Pull request by WhiteEgg**
New gateway directory ui

dependencies: create https://github.com/TRON-US/dir-index-html from https://github.com/WhiteEgg/dir-index-html/tree/btfs btfs branch for future update

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "assets/dir-index-html"]
 	path = assets/dir-index-html
-	url = https://github.com/ipfs/dir-index-html.git
+	url = https://github.com/TRON-US/dir-index-html.git
```
Comment (taiyangc) [inline .gitmodules:3]: This doesn't exist right now, where can we set it up? Should it just be a fork of ipfs/dir-index-html to start with?

Comment (WhiteEgg) [reply .gitmodules:3]: Yep, fork ipfs/dir-index-html and i will submit PR to that

Comment (WhiteEgg) [reply .gitmodules:3]: \* fork ipfs/dir-index-html and checkout v1.0.5 tag to master

Comment (taiyangc) [reply .gitmodules:3]: @WhiteEgg Updated here: https://github.com/TRON-US/dir-index-html please submit PR there.

Comment (WhiteEgg) [reply .gitmodules:3]: @taiyangc https://github.com/TRON-US/dir-index-html/pull/2

Comment (taiyangc) [reply .gitmodules:3]: merged, please update reference here

Comment (WhiteEgg) [reply .gitmodules:3]: @taiyangc update with force push

**Comment by taiyangc** [comment]
ok to test

**Review by taiyangc — APPROVED** [review]

**Pull request merged**